01 готов

VProkhorenko opened the pull request.

vvscode commented on 01/ht/VProkhorenko/script.js:
```diff
@@ -0,0 +1,137 @@
+ */
+function isPolindrom(textString) {
+   var textStringNew = textString.split("").reverse().join("");
+    if (textStringNew===textString)
```
нужен ли здесь `if` ?

VProkhorenko commented:
Действительно, можно обойтись без if.
А от промежуточной переменной тоже лучше избавиться?

function isPolindrom(textString) {
    return (textString === textString.split("").reverse().join(""));
}


19 октября 2017 г., 18:19 пользователь Vasiliy Vanchuk <
notifications@github.com> написал:

> *@vvscode* commented on this pull request.
> ------------------------------
>
> In 01/ht/VProkhorenko/script.js
> <https://github.com/vvscode/js--base-course/pull/140#discussion_r145733270>
> :
>
> > +            i % 5 === 0 && 'Buzz' || i);
> +    }
> +
> +}
> +
> +
> +/**
> + * реализовать фукнцию  `isPolindrom`,
> + * которая принимает на вход строку и возвращает результат проверки (`true`/ `false` ),
> + * является строка полндромом (одинакого читается с лева на право и с права на лево ) или нет
> + * @param {string} textString
> + * @return {boolean} Является строка полндромом (одинакого читается с лева на право и с права на лево ) или нет
> + */
> +function isPolindrom(textString) {
> +   var textStringNew = textString.split("").reverse().join("");
> +    if (textStringNew===textString)
>
> нужен ли здесь if ?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/vvscode/js--base-course/pull/140#pullrequestreview-70576775>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AZ8b_JesTmE6kCCYobaa8GHEsBf3r8haks5st2iHgaJpZM4P-85a>
> .
>

vvscode commented:
переменная - как желаешь
